internal/handlers/common/commands.go

```diff
@@ -32,10 +32,6 @@ type Command struct {
 // Commands is a map of commands that common.Handler interface can support.
 // Order of entries matches the interface definition.
 var Commands = map[string]Command{
-	"listCommands": {
-		Help:    "Returns a list of currently supported commands.",
-		Handler: (Handler).MsgListCommands,
```

Comment:
Handler for that command is in the same package, so we could wire it there, together with debug_error and debug_panic

Reply:
added with nil's